ie-comunicaciones.csl

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
I've removed the locale. But:
In Mendeley's last version you can choose the language to use.
I've been invited to publish here an article in english. So I guess they accept english too.